content/learning-paths/cross-platform/memory-latency/_index.md

```diff
@@ -1,35 +1,38 @@
 ---
-title: Memory Latency
+title: Memory latency for application software developers
 
 minutes_to_complete: 40
 
-who_is_this_for: This is an introductory topic for Arm developers who are interested in optimal dealing with memory latency and cache usage in their code
+who_is_this_for: This is an introductory topic for Arm developers who want learn about memory latency and cache usage in application programming. 
 
 learning_objectives: 
-    - Understand the importance of memory latency and how to reduce its impact
-    - Learn about cache alignment and how it can help
+    - Explain the importance of memory latency and how to reduce its impact
+    - Identify how cache alignment impacts performance
     - Use cache prefetching to improve performance
 
 prerequisites:
-    - An Arm computer running Linux OS and a recent version of compiler (Clang or GCC) installed
+    - An Arm computer running Linux with recent version of Clang or GCC installed.
 
-author_primary: Konstantinos Margaritis, VectorCamp
+author_primary: Konstantinos Margaritis
```

Comment:
In the previous LP, https://github.com/ArmDeveloperEcosystem/arm-learning-paths/blob/main/content/learning-paths/cross-platform/restrict-keyword-c99/_index.md?plain=1 the company name was left in, would it be possible to keep it in this one as well?

Reply:
@markos The company name will show up on the index page. The info is templated and the data comes from contributors.csv at the top of the repository. You can add more info such as website, Linkedin, etc. Please submit a new PR for just contributors.csv with any additional info you want for your name. Currently, it's just company name. Thanks.

Reply:
Should I do the same then for the restrict LP and remove the company name from there?